doc/tools/west/without-west.rst

```diff
@@ -0,0 +1,46 @@
+.. _no-west:
+
+Using Zephyr without west
+#########################
+
+There are many valid usecases to avoid using a meta-tool such as west which has
+been custom-designed for a particular project and its requirements.
+This section applies to Zephyr users who fall into one or more of these
+categories:
+
+- Already use a multi-repository management system (for example Google repo)
+
+- Do not need additional functionality beyond compiling and linking
+
+In order to obtain the Zephyr source code and build it without the help of
+west you will need to manually clone the repositories listed in the
+`manifest file`_ one by one, at the path indicated in it.
+
+.. code-block:: console
+
+   mkdir zephyrproject
+   cd zephyrproject
+   git clone https://github.com/zephyrproject-rtos/zephyr
+   # clone additional repositories listed in the manifest file
+
```

Comment:
And check out the revision specified in the manifest

Reply:
This is also just moving around old docs ;). Will add a fix.